drivers/src/java/scriptella/driver/csv/opencsv/CSVWriter.java

```diff
@@ -128,20 +166,28 @@ public void writeNext(String[] nextLine) throws IOException {
             String nextElement = nextLine[i];
             if (nextElement == null)
                 continue;
-            if (quotechar != NO_QUOTE_CHARACTER)
+
+            final Boolean hasSpecialCharacters =  nextElement.indexOf(quotechar) != -1 || nextElement.indexOf(escapechar) != -1 || nextElement.indexOf(separator) != -1 

```

Comment:
nit: Please make hasSpecialCharacters a primitive boolean and break the line at 120 characters or less.